examples/echoserver/echoserver.c

```diff
@@ -1743,6 +1865,7 @@ THREAD_RETURN WOLFSSH_THREAD echoserver_test(void* args)
             fprintf(stderr, "Couldn't allocate thread context data.\n");
             exit(EXIT_FAILURE);
         }
+        memset(threadCtx, 0, sizeof *threadCtx);
```

Comment:
Note: I see a mix of `memset` and `WMEMSET` in some of the examples. But looks like mainly using the straight stdlib calls.